src/Tools/dotnet-monitor/HostBuilderHelper.cs

```diff
@@ -112,6 +138,23 @@ public static IHostBuilder CreateHostBuilder(HostBuilderSettings settings)
                 });
         }
 
+        private static void UpdatePropertiesWarningList(HostBuilderContext context, string warningMessage)
+        {
+            if (context.Properties.TryGetValue(HostBuilderResults.ResultKey, out object resultsObject))
+            {
+                if (resultsObject is HostBuilderResults)
```

Comment:
nit: `resultsObject is HostBuilderResults hostBuilderResults`. No need for a secondary cast.